internal/services/config/config.go

```diff
@@ -257,6 +258,7 @@ var defaultConfig = Config{
 	Runservice: Runservice{
 		RunCacheExpireInterval:     7 * 24 * time.Hour,
 		RunWorkspaceExpireInterval: 7 * 24 * time.Hour,
+		RunLogExpireInterval:       7 * 24 * time.Hour,
```

Comment:
Should the logs by default last longer than workspaces?

Reply:
@sgotti How much time must I set in RunLogExpireInterval?